### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by eagletusk** [HashNuke/Python-Arduino-Prototyping-API#6]
Just thought your "sample_blink" would be better off as mimicking the the sample on blink bundled with Arduino.

while 1:
    my_board.setHigh(13)
    time.sleep(1)
    my_board.setLow(13)
    time.sleep(1)
    i+=1

not sure what happens with python if i rolls over, imagine it's similar to c but I don't know...

**Comment by HashNuke** [comment]
Agree. That'll give users some context. I'll test it out once and change it.
